shacl-compact-syntax/index.html

```diff
@@ -716,12 +723,76 @@ <h2>Grammar and Production Rules</h2>
 				The test cases are not normative.
 			</p>
 		</section>
+
+    <section class="appendix normative" id="iana-considerations">
+      <h2>IANA Considerations</h2>
+
+      <p>This section has been submitted to the Internet Engineering Steering
+        Group (IESG) for review, approval, and registration with IANA.</p>
+
+      <h3 id="text-shaclc">text/shaclc</h3>
+      <dl>
+        <dt>Type name:</dt>
+        <dd>text</dd>
+        <dt>Subtype name:</dt>
+        <dd>shaclc</dd>
+        <dt>Required parameters:</dt>
+        <dd>None</dd>
+        <dt>Encoding considerations:</dt>
+        <dd>SHACL Compact Syntax (SHACLC) is a text language encoded in UTF-8.</dd>
+        <dt>Security considerations:</dt>
+        <dd>
+          <p>
+            Revealing the structure of an RDF graph can reveal information about the content of conformant data.
+            For instance, a schema with a predicate to describe cancer stage indicates that conforming graphs describe patients with cancer.
+          </p>
+          <p>
+            The process of testing a graph's conformance to a schema could draw significant system resources and be a vector for Denial of Service attacks.
+          </p>
+          <p>
+            <a href="https://www.w3.org/TR/turtle/#sec-mediaReg">RDF Turtle security considerations</a> about IRI spoofing may also apply here.
+          </p>
+        </dd>
+        <dt>Interoperability considerations:</dt>
+        <dd>Not Applicable</dd>
+        <dt>Published specification:</dt>
+        <dd><a href="https://w3c.github.io/shacl/shacl-compact-syntax/">This document</a></dd>
+        <dt>Applications that use this media type:</dt>
+        <dd>An implementations of SHACLC is part of the <a href="https://github.com/TopQuadrant/shacl">TopQuadrant SHACL API</a>
+        </dd>
+        <dt>Additional information:</dt>
+        <dd>
+          <dl>
+            <dt>Magic number(s):</dt>
+            SHACLC documents will likely have the words <code>PREFIX</code> or <code>BASE</code> (case sensitive) near the beginning of the document.
+            However, the same words may appear in Turtle and SPARQL documents.
+            <dd></dd>
+            <dt>File extension(s):</dt>
+            <dd>.shaclc</dd>
```

Comment:
@HolgerKnublauch please add `.shc` as per Andy's request. I've added it in my IANA submission (IANA tracking #1172417)